generate-release/.gitignore

```diff
@@ -1,7 +1,4 @@
-target
-assets
-data/
```

Comment:
These folders don't seem to be used in `generate-release`. Perhaps this was copy-and-pasted from somewhere else?